conf/npm/generateentrypoints.js

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const { ALL_LANGUAGES } = require('../i18n/constants');
+
+/**
+ * Add entry points to css bundles to the export field
+ *
+ * @param {Object} exportJSON
+ * @param {string} lang
+ */
+function addCSSBundleEntries (exportJSON) {
+  const cssEntries = {
+    './css': './dist/answers.css',
+    './rtl-css': './dist/answers.rtl.css'
+  };
+  Object.assign(exportJSON.exports, cssEntries);
+}
+
+/**
+ * Add entry points to js and template bundles to the export field for default language en
+ *
+ * @param {Object} exportJSON
+ * @param {string} lang
+ */
+function addDefaultLanguageBundleEntries (exportJSON) {
```

Comment:
I'm not sure that we need a function for this and for 'addLanguageBundleEntries`. This could be an object and in appendExportsField we use the spread operator to add to the exports